common/src/main/java/com/thekillerbunny/goofyplugin/lua/GoofyAPI.java

```diff
@@ -262,6 +262,28 @@ public void loadAvatar(@LuaNotNil String playerUUID) {
         NetworkStuff.getUser(new UserData(uuid));
     }
 
+    @LuaWhitelist
+    @LuaMethodDoc(
+        overloads = {
+            @LuaMethodOverload(
+                argumentTypes = { String.class },
+                argumentNames = { "playerUUID" }
+            )
+        },
+        value = "goofy.get_player_color"
+    )
+    public String getAvatarColor(String playerUUID) {
+        UUID uuid = UUID.fromString(playerUUID);
```

Comment:
this can VM error iirc; blocking merge

Reply:
If you really don't want a VM error fix it yourself

Reply:
bet

Reply:
Make sure to fix it in the other functions as well :trol:

Reply:
I'll make a separate PR soon™.